Geometry/CommonDetUnit/interface/GeomDet.h

```diff
@@ -119,9 +125,10 @@ class GeomDet {
 
   ReferenceCountingPointer<Plane>  thePlane;
   DetId m_detId;
-  int m_index;
+  int m_index=-1;
```

Comment:
@VinInn - it would be nice to clean up the class code a bit: sort public, protected and private members, name the latter either with leading m_* or the*, indent the code consistently.